### PR TITLE
Updated other supported operating systems

### PR DESCRIPTION
**Pull request by kasiak-msft**
Added iOS to list of supported Other operating systems. Added hyperlinks to supporting documentation for each OS so that the reader can find out which versions of these are supported. Updated note below list to clarify that reader can find more details in the linked documentation.

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 1]
```diff
@@ -115,12 +115,13 @@ The hardware requirements for Defender for Endpoint on devices are the same for
 
 
 ### Other supported operating systems
-- Android
-- Linux
-- macOS
+- [macOS](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-mac)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:118]:
```suggestion
- [macOS](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-mac)
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 2]
```diff
@@ -115,12 +115,13 @@ The hardware requirements for Defender for Endpoint on devices are the same for
...
+- [Linux](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-linux)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:119]:
```suggestion
- [Linux](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-linux)
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 3]
```diff
@@ -115,12 +115,13 @@ The hardware requirements for Defender for Endpoint on devices are the same for
...
+- [Android](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-android)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:120]:
```suggestion
- [Android](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-android)
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 4]
```diff
@@ -115,12 +115,13 @@ The hardware requirements for Defender for Endpoint on devices are the same for
...
+- [iOS](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:121]:
```suggestion
- [iOS](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios)
```

File: windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md [hunk 5]
```diff
@@ -115,12 +115,13 @@ The hardware requirements for Defender for Endpoint on devices are the same for
...
 
 > [!NOTE]
-> You'll need to know the exact Linux distributions and versions of Android and macOS that are compatible with Defender for Endpoint for the integration to work.
+> The exact versions of macOS, Android, iOS, and Linux distributions that are compatible with Defender for Endpoint are in the above linked documentation.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/minimum-requirements.md:124]:
```suggestion
> Follow these links to find the exact versions of macOS, Android, iOS, and Linux distributions that are compatible with Defender for Endpoint.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by e0i** [comment]
@kasiak-msft According to the style guidelines,  `en-us` locale string must not be present in a URL. Please follow up with the recommendations above to proceed with the merge. Thank you for this contribution.

**Review by illfated — APPROVED** [review]
Nice update, looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
It seems that the source file for the "minimum requirements" article has bee relocated to the [Microsoft 365 documentation repository](https://github.com/MicrosoftDocs/microsoft-365-docs)

Additionally, The changes proposed in this PR appear in the article online.

Link to section: https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide#other-supported-operating-systems

Thanks.

**Pull request closed without merging**